src/components/Shop/Navbar.tsx

```diff
@@ -16,15 +20,22 @@ const Navbar: FC = () => {
         />
       </div>
       <div>
-        {user ? (
-          <span>
-            {user.first_name} | (icon) {user.balance}kr - 0 Ølcoins (icon)
-          </span>
-        ) : (
-          ""
-        )}
+        <IconContext.Provider value={{ color: "white", size: "20px" }}>
+          {user ? (
+            <span>
+              {user.first_name} | <GiWallet /> {user.balance}kr - 0 Ølcoins
+              <IoIosBeer />
+            </span>
+          ) : (
+            ""
+          )}
+        </IconContext.Provider>
       </div>
-      <div>(icon)</div>
+      <IconContext.Provider value={{ color: "black", size: "40px" }}>
```

Comment:
It probably should be white, as it fits better on the blue color. I used black in the figma as it was the only color i found haha.